cpp/tests/groupby/common/groupby_test_util.hpp

```diff
@@ -33,6 +35,7 @@ inline void test_single_agg(column_view const& keys,
                             column_view const& expect_vals,
                             std::unique_ptr<experimental::aggregation>&& agg,
                             bool ignore_null_keys = true,
+                            bool stable_order = false,
                             bool keys_are_sorted = false,
```

Comment:
I know this isn't in this PR, but `bool` in function interfaces are less desirable, especially back-to-back-to-back `bool`s. The call sites become very unreadable. Alternate is to use `enum class`

Reply:
I suppose before doing this, I should fix cudf API: https://github.com/rapidsai/cudf/blob/8e1f4a7be9ad0a00d744f8901c31f4eb3733ebe3/cpp/include/cudf/groupby.hpp#L103-L106